### ISSUE TITLE
Style guide

### ISSUE THREAD

**Issue by softwarespot** [sgaurav/understanding-es6#26]
Right now the examples are good, but the styles are a little all over the place. One approach would be to use a style guide and enforce it somehow e.g. gulp. May I recommend [Airbnb](https://github.com/airbnb/javascript) as the basis for writing examples.

Another question, wouldn't it be good to use const/let instead of var in the examples?

**Comment by josokinas** [comment]
Agree with you...

But aren't there cases where you specifically need const, var, let? I see all of those to be in use with ES6, not leaving anything behind.

**Comment by softwarespot** [comment]
According to Airbnb's guide, const should always be used unless there are cases for let.

**Comment by josokinas** [comment]
https://github.com/airbnb/javascript#2.1 - you're right.

**Comment by softwarespot** [comment]
Of course it's a style guide, so open to a lot of interpretation. Though I notice more and more people (including myself) are adopting it these days.

**Comment by sgaurav** [comment]
I agree with you, this whole thing started as me learning and in the process creating help of everyone else. That is reason styles are a bit messed up.

I will update this thing sometime over this week else please send a PR and I will merge.

**Comment by softwarespot** [comment]
I can do it tonight, that's no problem
